aarone_backend/contacts/tasks.py

```diff
@@ -0,0 +1,33 @@
+from celery import shared_task
+from django.template.loader import render_to_string
+from django.core.mail import EmailMessage
+from django.conf import settings
+
+@shared_task
+def send_email_task(instance_name, instance_email, instance_query):
+    # Send email notification with email template
+    subject = "Thanks for your message, we'll get back to you soon."
+    name_parts = instance_name.split()
+    first_name = "customer"
+    if name_parts:  # Check if name_parts is not empty
+        first_element = name_parts[0]  # Get the first element
+        if first_element and first_element[0].islower():  # Check if the first character is lowercase
+            # Capitalize the first character
+            first_name = first_element[0].upper() + first_element[1:]
+    html_content = render_to_string('new-email.html', {'name': first_name, 'email': instance_email, 'query': instance_query})
+    from_email = settings.DEFAULT_FROM_EMAIL
```

Comment:
[nit] We use the `DEFAULT_FROM_EMAIL` here, maybe we should update the variable name?